model-providers/ollama/runtime/src/main/java/io/quarkiverse/langchain4j/ollama/runtime/config/LangChain4jOllamaConfig.java

```diff
@@ -69,6 +69,12 @@ interface OllamaConfig {
         @WithDefault("true")
         Boolean enableIntegration();
 
+        /**
+         * Whether to enable the experimental tools
+         */
+        @WithDefault("false")
+        Optional<Boolean> experimentalTools();
```

Comment:
Shouldn't we rather just name it `tools` and mark it as experimental in a comment? To avoid having to do a breaking change once we don't consider it experimental anymore...

Reply:
I would name it `enableTools`

Reply:
I think this will stay experimental till ollama implements tools feature.